RELEASES.md

```diff
@@ -96,7 +96,8 @@ backports until the end of life date. They may also accept a wider range of
 patches than non-_LTS_ releases to support the longer term maintainability of the
 branch, including library dependency, toolchain (including Go) and other version updates
 which are needed to ensure each release is built with fully supported dependencies and
-remains usable by containerd clients. There should be at least a 6-month overlap between
+remains usable by containerd clients. _LTS_ releases can also accept feature backports
+to support new Kubernetes releases. There should be at least a 6-month overlap between
```

Comment:
The default action has to be reject it though, for long-term stability,
This is still negotiable when the feature is a hard dependency for a new release of kube.

Reply:
Updated as suggested. Please, review again.